# Header with timestamp and frame_id
Header header

# Id of camera
uint32 camId

# List of detections
Detection[] detections